modules/ephemeris/src/main/scala/gem/horizons/HorizonsClient.scala

```diff
@@ -33,11 +33,8 @@ object HorizonsClient {
   /** Horizons permits only one request at a time from a given host, so all
     * requests should go through this client.
     */
-  val client: Client[IO] =
-    PooledHttp1Client[IO](
-      maxTotalConnections = 1,
-      config = BlazeClientConfig.defaultConfig.copy(responseHeaderTimeout = 20.seconds)
```

Comment:
Will there still be 1 `maxTotalConnections`?  I had specified that in hopes of complying with the JPL limit on connections from a single client.

Reply:
Ah, I don't think so. I will try to figure that out.